app/models/search/work_indexer.rb

```diff
@@ -4,6 +4,15 @@ def self.klass
     "Work"
   end
 
+  def self.index_all(options={})
```

Comment:
Instead of overriding `create_index` you could override `settings`.

In the parent class:

https://github.com/otwcode/otwarchive/blob/a5487041e1e064521262ba34ef14d814b67ad8cc/app/models/search/indexer.rb#L40-L50

So in this subclass, settings can return `index: { number_of_shards: 9 }`, and we switch `merge` to `deep_merge` in the parent class.

Reply:
On second thoughts, doing so will break the shards override for `create_index`.